### PR TITLE
translate four pingcap/docs-cn PRs

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

The changes in this PR are the English version of the following PRs:

- https://github.com/pingcap/docs-cn/pull/12181
- https://github.com/pingcap/docs-cn/pull/12145
- https://github.com/pingcap/docs-cn/pull/12088
- https://github.com/pingcap/docs-cn/pull/13504

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12181, https://github.com/pingcap/docs-cn/pull/12145, https://github.com/pingcap/docs-cn/pull/12088, https://github.com/pingcap/docs-cn/pull/13504
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.4.0.md [hunk 1]
```diff
@@ -132,6 +132,10 @@ In v6.4.0-DMR, the key new features and improvements are as follows:
 
     For more information, see [User document](/system-variables.md#tidb_stats_load_sync_wait-new-in-v540).
 
+* Reduce the impact of batch write requests on the response time of lightweight transaction writes [#13313](https://github.com/tikv/tikv/issues/13313) @[glorv](https://github.com/glorv)
+
+    The business logic of some systems requires periodic batch DML tasks, but processing these batch write tasks can increase the latency of online transactions. In v6.3.0, TiKV optimizes the priority of read requests in hybrid workload scenarios, so you can enable the [`readpool.unified.auto-adjust-pool-size`](/tikv-configuration-file.md#auto-adjust-pool-size-new-in-v630) configuration item to let TiKV automatically adjust the size of the UnifyReadPool thread pool for all read requests. In v6.4.0, TiKV can also dynamically identify and prioritize write requests, and control the maximum number of bytes that the Apply thread can write for one FSM (Finite-state Machine) in one round of poll, thus reducing the impact of batch write requests on the response time of transaction writes.
```
Comment (songrijie) [inline releases/release-6.4.0.md:137]:
```suggestion
    The business logic of some systems requires periodic batch DML tasks, but processing these batch write tasks increases the latency of online transactions. In v6.3.0, TiKV optimizes the scheduling of read requests in hybrid workload scenarios, so you can enable the [`readpool.unified.auto-adjust-pool-size`](/tikv-configuration-file.md#auto-adjust-pool-size-new-in-v630) in configuration to let TiKV automatically adjust the size of the UnifyReadPool thread pool for all read requests. In v6.4.0, TiKV can also dynamically identify and prioritize write requests, and control the maximum bytes that the Apply thread can write for one FSM (Finite-state Machine) in one round of poll, thus reducing the impact of batch write requests on the response time of transactional writes.
```

File: releases/release-6.4.0.md [hunk 2]
```diff
@@ -132,6 +132,10 @@ In v6.4.0-DMR, the key new features and improvements are as follows:
 
     For more information, see [User document](/system-variables.md#tidb_stats_load_sync_wait-new-in-v540).
 
+* Reduce the impact of batch write requests on the response time of lightweight transaction writes [#13313](https://github.com/tikv/tikv/issues/13313) @[glorv](https://github.com/glorv)
```
Comment (qiancai) [inline releases/release-6.4.0.md:135]:
```suggestion
* Reduce the impact of batch write requests on the response time of lightweight transactional writes [#13313](https://github.com/tikv/tikv/issues/13313) @[glorv](https://github.com/glorv)
```

**Review by hehechen — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -3886,7 +3886,8 @@ SHOW WARNINGS;
 - Default value: `80%`
 - Range:
     - You can set the value in the percentage format, which means the percentage of the memory usage relative to the total memory. The value range is `[1%, 99%]`.
-    - You can also set the value in memory size in bytes. The value range is `[0, 9223372036854775807]`. The memory format with the units "KB|MB|GB|TB" is supported. The `0` value means no memory limit.
+    - You can also set the value in memory size. The value range is `0` and  `[536870912, 9223372036854775807]` in bytes. The memory format with the units "KB|MB|GB|TB" is supported. `0` means no memory limit.
```
Comment (Oreoxmt) [inline system-variables.md:3889]:
```suggestion
    - You can also set the value in memory size. The value range is `0` and `[536870912, 9223372036854775807]` in bytes. The memory format with the units "KB|MB|GB|TB" is supported. `0` means no memory limit.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -388,6 +388,22 @@ Configuration items related to security.
...
++ Default value: ""
+
+### `session-token-signing-key` <span class="version-mark">New in v6.4.0</span>
+
+> **Warning:**
+>
+> The feature controlled by this parameter is under development. **Do not modify the default value**.
+
++ Default value: ""
+
```
Comment (djshow832) [inline tidb-configuration-file.md:406]: LGTM

**Pull request merged**